aks.go

```diff
@@ -0,0 +1,173 @@
+package main
```

Comment:
nit: this and `aks_test.go` are the only files/variables that use `aks` instead of `azure` and this code could work for both `aks` and self-hosted k8s clusters on Azure. Could you rename this to be `azure.go` instead?